containers/search/Searchbox.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { c } from 'ttag';
+import PropTypes from 'prop-types';
+import { SearchInput, Icon } from 'react-components';
+
+import { classnames } from '../../helpers/component';
+
+const Searchbox = ({ className = '', placeholder = '', value = '', onSearch }) => {
+    return (
+        <div className={classnames(['searchbox-container relative flex-item-centered-vert', className])}>
```

Comment:
Can we use a `form` here? And then use `onSubmit={onSearch}` and `type=submit` on `searchbox-search-button`.